app/javascript/components/ExpressDonation/ExpressDonation.js

```diff
@@ -64,6 +77,7 @@ export class ExpressDonation extends Component<Props, State> {
         // form will have the user's submitted values
         ...this.props.fundraiser.form,
       },
+      allow_duplicate: this.state.opt_for_redonation,
```

Comment:
Use consistent `camelCase` naming for JavaScript variable names: `optForRedonation` and `allowDuplicate` in this case.